include/hw_constants_checks.h

```diff
@@ -17,12 +16,20 @@ namespace gbt{
      *  \param[in] phase Phase value to check.
      *  \return Returns `false` in case of success; `true` in case of error. The precise error is logged and written to the `error` RPC key.
      */
-    inline bool checkPhase(wisc::RPCMsg *response, uint8_t phase){
-        if (phase < gbt::PHASE_MIN)
-            EMIT_RPC_ERROR(response, stdsprintf("The phase parameter supplied (%hhu) is smaller than the minimal phase (%hhu).", phase, gbt::PHASE_MIN), true)
-        if (phase > gbt::PHASE_MAX)
-            EMIT_RPC_ERROR(response, stdsprintf("The phase parameter supplied (%hhu) is bigger than the maximal phase (%hhu).", phase, gbt::PHASE_MAX), true)
-        return false;
+    inline void checkPhase(uint8_t phase){
+        if (phase < PHASE_MIN)
```

Comment:
brace vs whitespace delimited scopes (`c/c++` vs `python`), only the very next line is within the scope of this `if` statement